packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -286,10 +284,15 @@ type SingleClickOnDiagramElementTool implements Tool {
   label: String!
   iconURL: [String!]!
   appliesToDiagramRoot: Boolean!
-  selectionDescriptionId: String
+  dialog: Dialog
   targetDescriptions: [DiagramElementDescription!]!
 }
 
+type Dialog {
+  dialogDescriptionId: String!
+  dialogDescriptionType: String!
```

Comment:
- This should be a XxxDescription or be removed entirely since I'm not sure why it needs to be a dedicated type?
- You could use `id` and `type`
- I'm not sure why this concept isn't an interface and why you need more than `id`?

Reply:
The id currently contains only an UUID that does not allow to identify the dialog type.
If we want to keep only an id, we need to modify the way we generate this id to own the dialog type information?

Reply:
done